lerna.json

```diff
@@ -0,0 +1,10 @@
+{
+  "version": "2.6.1-SNAPSHOT",
```

Comment:
Do we plan to leave the versions as "2.6.1-SNAPSHOT" for now? I guess there's not really any reason to manually bump them now 🙂 We can wait until the 2.7.0 release PR to do that.

Reply:
> LGTM, will approve once failing tests are addressed.

Just pushed the fix for this 👍

Reply:
@t1m0thyj as long as it doesn't change anything when merged I wouldn't worry about it atm.